plugins/modelposepublisher/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2018 Istituto Italiano di Tecnologia 
+# Authors: see AUTHORS file.
```

Comment:
Please add yourself to the AUTHORS file, thanks!

Reply:
Done!